python/lsst/pipe/tasks/processCcd.py

```diff
@@ -12,30 +12,56 @@
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
-# see <http://www.lsstcorp.org/LegalNotices/>.
+# see <https://www.lsstcorp.org/LegalNotices/>.
 #
 from .processImage import ProcessImageTask
 from lsst.ip.isr import IsrTask
+from lsst.pipe.tasks.calibrate import CalibrateTask
+import lsst.afw.geom as afwGeom
+import lsst.afw.math as afwMath
 import lsst.afw.table as afwTable
 import lsst.pex.config as pexConfig
 import lsst.pipe.base as pipeBase
 
 class ProcessCcdConfig(ProcessImageTask.ConfigClass):
     """Config for ProcessCcd"""
-    doIsr = pexConfig.Field(dtype=bool, default=True, doc = "Perform ISR?")
+    doIsr = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc = "Perform ISR?"
+    )
     isr = pexConfig.ConfigurableField(
         target = IsrTask,
         doc = "Instrumental Signature Removal",
     )
+    doCalibrate = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc = "Perform calibration?"
+    )
+    doWriteCalibrate = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc = "Write calibration results?"
+    )
+    doWriteCalibrateMatches = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc = "Write icSrc to reference matches?"
+    )
```

Comment:
I'm happy to maintain consistency by changing all occurrences to the desired format :)